package.json

```diff
@@ -8,7 +8,8 @@
     "test-jerry": "deps/jerry/tools/run-tests.py --unittests",
     "test-iotjs": "tools/build.py --run-test --no-check-valgrind",
     "build": "tools/build.py",
-    "rebuild": "tools/build.py --clean"
+    "rebuild": "tools/build.py --clean",
+    "myrebuild": "tools/build.py --clean --external-include-dir /usr/local/Cellar/dbus/1.12.10/include/dbus-1.0 --external-include-dir /usr/local/lib/dbus-1.0/include"
```

Comment:
Remove this line

Reply:
I am sorry, I push the test file! Awkward!!!